### PR TITLE
Fix workflows

### LINKED ISSUE #4247: Replace travis with GH Workflows
Travis seems not running for few last commits. I guess GH workflows woild be more robust decision.

I can create PR.

### PR DESCRIPTION
**Pull request by ogonkov**
Fixes #4247

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -23,17 +24,17 @@ jobs:
       - uses: actions/setup-node@v2
         with:
           node-version: 6
-          cache: 'npm'
```
Comment (jgonggrijp) [inline .github/workflows/tests.yml:26]: Would it be possible to put back this line, or is it somehow preventing the whole thing from working?

**Review by jgonggrijp — COMMENTED** [review]
Thanks for your hard work @ogonkov!

I seem to recall that secrets are only available when the person triggering the CI (i.e., you in this case) has the right to access them. That might explain why Sauce is being skipped in this PR; we will see after merging whether this is indeed the (main) reason.

I did notice one error message in advance of the PhantomJS test run, about installing the `karma-sauce-launcher`. This might still be worth investigating, if you haven't already done so. https://github.com/jashkenas/backbone/pull/4251/checks#step:5:10

File: .github/workflows/tests.yml (hunk 1 above)
Comment (ogonkov) [reply .github/workflows/tests.yml:26]: It actually looking for lock file, so it would be possible after package-lock commit

**Comment by ogonkov** [comment]
All other vars, that i use is also empty. 

```
Run echo "Value: $GITHUB_RUN_NUMBER"
  echo "Value: $GITHUB_RUN_NUMBER"
  printenv
  shell: /usr/bin/bash -e {0}
  env:
    NPM_CONFIG_PROGRESS: false
    SAUCE_USERNAME: 
    SAUCE_ACCESS_KEY: 
    BUILD_NUMBER: $GITHUB_RUN_NUMBER-foobar
    BUILD_ID: 
    JOB_NUMBER: 
Value: 38
```

BUILD_ID and JOB_NUMBER is not secret, but also missing.

**Pull request merged**